Options.py

```diff
@@ -765,7 +766,7 @@ class VerifyKeys(metaclass=FreezeValidKeys):
     value: typing.Any
 
     @classmethod
-    def verify_keys(cls, data: typing.List[str]):
+    def verify_keys(cls, data: typing.Iterable[str]):
```

Comment:
I think adding the return type here wouldn't hurt when we touch this line?
```suggestion
    def verify_keys(cls, data: typing.Iterable[str]) -> None:
```

Reply:
I personally live with the mantra that no explicit return type means return type is None, but I will gladly add it.

Reply:
> no explicit return type means return type is None

I wish that were more accepted by mypy. The `-> None` can add a lot of clutter.
But, depending on the settings mypy will either complain that it's not there or just not check that function.

https://github.com/python/mypy/issues/9413